com.github.btpf.alexandria.json

```diff
@@ -0,0 +1,51 @@
+{
+  "id": "com.github.btpf.alexandria",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "42",
```

Comment:
Tauri?

Are you able to use the 2.0 alphas that support newer webkitgtk? This runtime is EOL.